### PR TITLE
fix: file name is not unique enough when running tests in parallel

### PR DESCRIPTION
**Pull request by kobenguyent**
## Motivation/Description of the PR
- The fileName is not unique when running tests in parallel and it leads to the issue that attachment couldn't attach to allure report.

Applicable helpers:
- [ ] Playwright

## Type of change
- [ ] :bug: Bug fix

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Comment by EgorBodnar** [comment]
a good fix, as a result of our discussion :raised_hands:

The custom attachment `fileName` should be unique for reporters or it would be rewritten

File: lib/helper/Playwright.js [hunk 1]
```diff
@@ -3122,7 +3122,19 @@ async function saveVideoForPage(page, name) {
 async function saveTraceForContext(context, name) {
   if (!context) return;
   if (!context.tracing) return;
-  const fileName = `${`${global.output_dir}${pathSeparator}trace${pathSeparator}${Date.now()}_${clearString(name)}`.slice(0, 245)}.zip`;
+  const fileName = `${`${global.output_dir}${pathSeparator}trace${pathSeparator}${makeId(20)}_${clearString(name)}`.slice(0, 245)}.zip`;
```
Comment (EgorBodnar) [inline lib/helper/Playwright.js]: Should the `fileName` be unique for one test or it probably should be unique within one test step too?

* If it should be unique per test, use `mocha current test id` or `nodejs worker.threadId`

* If it should be unique within one test use already installed CodeceptJS dependency - `uuid` to generate really unique id. 

So better to delete the new `function makeId(length) {}`

**Review by EgorBodnar — CHANGES_REQUESTED** [review]
Should the `fileName` be unique for one test or it probably should be unique within one test step too?

* If it should be unique per test, use `mocha current test id` or `nodejs worker.threadId`

* If it should be unique within one test use already installed CodeceptJS dependency - `uuid` to generate really unique id. 

So better to delete the new `function makeId(length) {}`

**Comment by kobenguyent** [comment]
> Should the `fileName` be unique for one test or it probably should be unique within one test step too?
> 
> * If it should be unique per test, use `mocha current test id` or `nodejs worker.threadId`
> * If it should be unique within one test use already installed CodeceptJS dependency - `uuid` to generate really unique id.
> 
> So better to delete the new `function makeId(length) {}`

updated per CR. Thanks for pointing it out!

**Review by EgorBodnar — APPROVED** [review]

**Review by Arhell — APPROVED** [review]

**Pull request merged**